tests/Profile.spec.ts

```diff
@@ -65,391 +71,565 @@ test.describe(`Testfälle für das eigene Profil anzeigen: Umgebung: ${process.e
+          // 2FA
+          // Aktuell wird der Abschnitt 2FA generell nicht angezeigt
+          // await expect(profileView.cardHeadline_2FA).toHaveText('Zwei-Faktor-Authentifizierung');
+          // await expect(profileView.button_2FAEinrichten).toBeEnabled();
+        }
+      });
+    }
+  );
+
+  test(
+    'Das eigene Profil öffnen, Passwort Ändern öffnen, und Status des Benutzernamenfelds prüfen',
+    { tag: [LONG, STAGE] },
+    async ({ page }) => {
+      const profileView = new ProfilePage(page);
+      const header = new HeaderPage(page);
+      const login = new LoginPage(page);
+
+      const organisation = testschule;
+      const rollenart = 'LERN';
+
+      await test.step(`Lehrer via api anlegen und mit diesem anmelden`, async () => {
+        const idSP = await getSPId(page, 'itslearning');
+        const userInfo: UserInfo = await createRolleAndPersonWithUserContext(
+          page,
+          organisation,
+          rollenart,
+          await generateNachname(),
+          await generateVorname(),
+          idSP,
+          await generateRolleName()
+        );
+        roleId.push(userInfo.rolleId);
+        username.push(userInfo.username);
+
+        await header.logout();
+        await header.button_login.click();
+        await login.login(userInfo.username, userInfo.password);
+        await login.UpdatePW();
+      });
+
+      await test.step(`Profil öffnen`, async () => {
+        await header.button_profil.click();
+      });
+
+      await test.step(`Passwort Ändern öffnen`, async () => {
+        await expect(profileView.button_ZurueckVorherigeSeite).toBeVisible();
+        await expect(profileView.text_h2_Ueberschrift).toHaveText('Mein Profil');
+        // Passwort
+        await expect(profileView.cardHeadline_Passwort).toHaveText('Passwort');
+        await expect(profileView.button_NeuesPasswortSetzen).toBeEnabled();
 
-    await test.step(`Passwort Ändern öffnen`, async () => {
-      await expect(profileView.button_ZurueckVorherigeSeite).toBeVisible();
-      await expect(profileView.text_h2_Ueberschrift).toHaveText('Mein Profil');
-      // Passwort
-      await expect(profileView.cardHeadline_Passwort).toHaveText('Passwort');
-      await expect(profileView.button_NeuesPasswortSetzen).toBeEnabled();
+        profileView.button_NeuesPasswortSetzen.click();
+        profileView.button_PasswortAendern.click();
+      });
+
+      await test.step(`Status des Benutzernamenfelds prüfen`, async () => {
+        await expect(profileView.label_username).toHaveText(username[0]); // Benutzername ist nicht änderbar weil es nur Text ist
+        await expect(profileView.text_p_LoginPrompt).toHaveText('Bitte geben Sie Ihr aktuelles Passwort ein.');
+        await expect(profileView.input_password).toBeEnabled();
+        await page.goBack();
+      });
+    }
+  );
+
+  test(
+    'Das eigene Profil öffnen, 2FA Einrichten öffnen und Einrichtung soweit möglich',
+    { tag: [LONG, STAGE] },
```

Comment:
I recommend to put the test in the testsuite LONG only,  because we need a proof for the stability before running in testsuite STAGE.